bay-services/osa-probable-vulnerability-inference-job.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: none
```

Comment:
Does it not need a hash?

Reply:
As of now it is targeted only for staging and it doesn't need a hash.